.cirrus.yml

```diff
@@ -27,3 +27,26 @@ task:
     - gmake check ${MAKEJOBS} VERBOSE=1
   functional_test_script:
     - ./test/functional/test_runner.py --jobs 9 --ci --extended --exclude feature_dbcrash --combinedlogslen=1000 --quiet --failfast
+task:
```

Comment:
Duplication of key `task` in mapping?

Reply:
Yes, this is the correct syntax:

https://cirrus-ci.org/guide/writing-tasks/#task-execution-dependencies